src/runtime/particle-execution-context.ts

```diff
@@ -28,14 +27,13 @@ import {WasmContainer, WasmParticle} from './wasm.js';
 import {Dictionary} from './hot.js';
 import {UserException} from './arc-exceptions.js';
 import {Store} from './store.js';
-import {Flags} from './flags.js';
 import {SystemTrace} from '../tracelib/systrace.js';
 import {delegateSystemTraceApis} from '../tracelib/systrace-helpers.js';
 import {ChannelConstructor} from './channel-constructor.js';
 import {Ttl} from './recipe/ttl.js';
 
 export type PecFactory = (pecId: Id, idGenerator: IdGenerator) => MessagePort;
-type UnifiedStorageProxy = Store|StorageProxyNG<CRDTTypeRecord>;
+type UnifiedStorageProxy = Store|StorageProxy<CRDTTypeRecord>;
```

Comment:
we should be able to use StorageProxy<CRDTTypeRecord> instead of UnifiedStorageProxy now